### ISSUE TITLE
newAccount returning status as empty string

### ISSUE THREAD

**Issue by rmbolger** [letsencrypt/pebble#90]
Currently testing against commit eb01c4f using Windows PowerShell 5.1's `Invoke-WebRequest`.

When I submit a newAccount request to the `/sign-me-up` endpoint, everything goes as expected except the value of `status` in the returned account object. The HTTP response code is 201. The headers include a new nonce and Location Url of the new account. And the body is returned as:

```javascript
{
   "status": "",
   "contact": [
      "mailto:me@example.com"
   ],
   "termsOfServiceAgreed": true,
   "onlyReturnExisting": false
}
```

[Section 7.3 of the 09 spec](https://tools.ietf.org/html/draft-ietf-acme-acme-09#section-7.3) would have me believe `status` should be set to `"valid"`. Or am I misinterpreting?

Here are the associated Pebble logs:

```
Pebble 2018/02/07 14:48:07 Generated new root issuer with serial 7b3527e7d7cfcc40
Pebble 2018/02/07 14:48:07 Generated new intermediate issuer with serial 160a8981343113e1
Pebble 2018/02/07 14:48:07 Configured to reject 15% of good nonces
Pebble 2018/02/07 14:48:07 Pebble running, listening on: 0.0.0.0:14000
Pebble 2018/02/07 14:48:23 HEAD /nonce-plz -> calling handler()
Pebble 2018/02/07 14:52:23 POST /sign-me-up -> calling handler()
Pebble 2018/02/07 14:52:23 There are now 1 accounts in memory
```

**Comment by cpu** [comment]
Hi @rmbolger 

> Section 7.3 of the 09 spec would have me believe status should be set to "valid". Or am I misinterpreting?

You're not misinterpreting. Sounds like a Pebble bug. Thanks for reporting!

**Comment by cpu** [comment]
Hi again @rmbolger :wave:

This should be fixed for you in master with 48220a83e693ea77ccadbba76a44ca74ef4fa6f4. Thanks again

**Comment by rmbolger** [comment]
Thanks everyone.
